packages/webapp/src/containers/Finances/AddSale/index.jsx

```diff
@@ -37,10 +37,9 @@ function AddSale() {
   const revenueType = useSelector(revenueTypeByIdSelector(revenue_type_id));
   const revenueTypes = useSortedRevenueTypes();
   const revenueTypeReactSelectOptions = mapRevenueTypesToReactSelectOptions(revenueTypes);
-  const { revenue_name, revenue_translation_key, farm_id } = revenueType;
-  const translatedRevenueName = farm_id
-    ? revenue_name
-    : t(`revenue:${revenue_translation_key}.REVENUE_NAME`);
+  const translatedRevenueName = revenueType?.farm_id
```

Comment:
This is definitely the code that's crashing, but unfortunately with just the optional chaining check I'm seeing this then on the Add Sale title:

![Screenshot 2023-11-10 at 2 46 44 PM](https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/33043357-bbea-4772-8a65-b9d102534f47)

Reply:
You are right! Just fixed this